inc/class_parser.php

```diff
@@ -1592,6 +1595,11 @@ function mycode_auto_url($message)
 	{
 		$message = " ".$message;
 
+		if($this->options['allow_url'] != 1)
+		{
+			return $message;
+		}
```

Comment:
Hi Ben, I think we can keep this, no need to add it to every parser build instance, developers could still use it even if the core doesn't.

However, I think it should be added in the following list:
https://github.com/mybb/mybb/blob/5b6c198abc6a21906363a4a215f075a7e06f2277/inc/class_parser.php#L13-L21

Ideally that piece of code should describe each option, but that is not part of this PR. @euantorano

Reply:
Thanks @Sama34 @euantorano I have applied it in the commented note above